src/components/App/index.js

```diff
@@ -103,9 +104,20 @@ class ModalApp extends Component {
         this.onInvalidJWT('hideOnfidoLogo feature not enabled for this account.')
       }
 
-    } else if (!options.mobileFlow && token && token !== prevToken) {
+    } else if (!options.mobileFlow && isNewToken) {
       this.props.actions.hideOnfidoLogo(false)
     }
+    if (options.enterpriseFeatures?.cobrand && !isHideOnfidoLogoEnabled && isNewToken) {
+      const validEnterpriseFeatures = getEnterpriseFeaturesFromJWT(token)
+      if (validEnterpriseFeatures?.cobrand) {
+        this.props.actions.showCobranding(options.enterpriseFeatures.cobrand)
+      } else {
+        this.props.actions.showCobranding(null)
+        const errorMessage = 'EnterpriseFeatureNotEnabledException: Enterprise feature cobrand not enabled for this account.'
+        Tracker.trackException(errorMessage)
+        this.onInvalidJWT(errorMessage)
```

Comment:
If the JWT is invalid, by the time this logic is run, the host app would have already received the onError error callback so I don't see this why we need to send it again. Maybe we need return enterprise features errors in the onError callback...

Reply:
I've split the enterprise feature errors into a separate error handler. This should make it clearer which type of JWT error is being called.